ocpp/v201/enums.py

```diff
@@ -721,7 +741,7 @@ class HashAlgorithmType(StrEnum):
     sha512 = "SHA512"
 
 
-class IdTokenType(StrEnum):
+class IdTokenEnumType(StrEnum):
```

Comment:
Can you help me understand why are we renaming `IdTokenType` and `UnitOfMeasureType` to `IdTokenEnumType` and `StandardizedUnitsOfMeasureType`?

Reply:
This is related to OCPP nomenclature